src/Payments.Mvc/package.json

```diff
@@ -25,6 +25,7 @@
     "react": "16.3.2",
     "react-dom": "16.3.2",
     "react-hot-loader": "4.1.0",
+    "node-sass": "^4.7.2",
```

Comment:
@jSylvestre why would any of this require using node-sass?

Reply:
@srkirkland This doesn't require it, but it does fix a build issue on AppVeyor.